docs/pages/stacks.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Remix Stacks
+description: The quickest way to get rocking and rolling with Remix
+order: 3
+---
+
+# Remix Stacks
+
+Remix Stacks is a feature of the Remix CLI that allows you to generate a Remix project quickly and easily. There are several built-in and official stacks that are full blown applications. You can also make your own (read more below).
+
+The built-in official stacks come ready with common things you need for a production application including:
+
+- Database
+- Automatic deployment pipelines
+- Authentication
+- Testing
+- Linting/Formatting/TypeScript
+
+What you're left with is everything completely set up for you to just get to work building whatever amazing web experience you want to build with Remix. Here are the built-in official stacks:
+
+- [The Blues Stack](https://github.com/remix-run/blues-stack): Deployed to the edge (distributed) with a long-running Node.js server and PostgreSQL database. Intended for large and fast production-grade applications serving millions of users.
+- [The Indie Stack](https://github.com/remix-run/indie-stack): Deployed to a long-running Node.js server with a persistent SQLite database. Intended to get you going quickly with little complexity for prototypes and proof of concepts that can be later updated to the Blues stack without much trouble.
+- [The Grunge Stack](https://github.com/remix-run/grunge-stack): Deployed to a serverless function running Node.js with DynamoDB for persistance. Intended for folks who want to deploy a production-grade application on AWS infrastructure serving millions of users.
+
+Yes, these are named after music genres. 🤘 Rock on.
+
+There will be more stacks available in the future. And you can make your own (and we strongly encourage it)!
+
+## Custom Stacks
+
+The Remix CLI will help you get started with one of these built-in stacks, but if you want, you can create your own stack and the Remix CLI will help you get started with that stack. There are several ways to do this, but the most straightforward is to create a GitHub repo:
+
+```
+npx create-remix@latest --template my-username/my-repo
+```
+
+Custom stacks give an enormous amount of power and flexibility and we hope you create your own that suites the preferences of you and your organization (feel free to fork ours!).
+
+### `--template`
+
+The template option can be any of the following values:
+
+- The name of a stack in the remix-run GH org (e.g. `blues-stack`)
+- A GH username/repo combo (e.g. `mcansh/snkrs`)
+- A file path to a directory on disk (e.g. `/my/remix-stack`)
+- A path to a tarball on disk (e.g. `/my/remix-stack.tar.gz`)
+- A URL to a tarball (e.g. `https://example.com/remix-stack.tar.gz`)
+- A file URL (e.g. `file:///Users/michael/remix-stack.tar.gz`)
+
+Additionally, if your stack is in a private GitHub repo, you can set a GitHub token in the `PRIVATE_GITHUB_TOKEN` environment variable:
```

Comment:
i currently have this as just `GITHUB_TOKEN`, but i have no preference on name

Reply:
I looked in the commit where you added it and it said `PRIVATE_GITHUB_TOKEN`, but I think `GITHUB_TOKEN` is better so I'll change this 👍